tests/e2e/specs/books-navigation.js

```diff
@@ -2,26 +2,26 @@
 // http://nightwatchjs.org/guide#usage
 
 module.exports = {
-  'load home': (browser) => {
+  'load home': browser => {
     browser
       .url(process.env.VUE_DEV_SERVER_URL)
       .waitForElementVisible('#app', 5000)
       .assert.elementPresent('.welcome')
-      .assert.containsText('h1', 'Welcome to Vuetify')
+      .assert.containsText('h1', 'Welcome to Vuetify');
   },
-  'navigate to books': (browser) => {
+  'navigate to books': browser => {
     browser
       .click('#toolbar-books-button')
       .waitForElementVisible('#books-container', 5000)
-      .assert.elementPresent('input')
+      .assert.elementPresent('input');
   },
-  'search books using game as title': (browser) => {
+  'search books using game as title': browser => {
     browser
       .setValue('input[type=text]', 'game')
       .pause(5000)
-      .getLogTypes((result) => console.log(result))
-      .getLog('browser', (logEntriesArray)=> console.log(logEntriesArray))
+      .getLogTypes(result => console.log(result))
+      .getLog('browser', logEntriesArray => console.log(logEntriesArray))
```

Comment:
Oh, now that you are at it remove these logs, I forgot to remove them in my PR xD